README.md

```diff
@@ -1,10 +1,10 @@
-phpspec-code-coverage
-=====================
+# phpspec-code-coverage
+
```

Comment:
I changed this because the Scrutinizer test was stuck and needed a change to push something new.

I can revert this if you want.